packages/react-next/src/components/FocusTrapZone/examples/FocusTrapZone.Box.Click.Example.tsx

```diff
@@ -6,10 +6,20 @@ import { Toggle, IToggle } from '@fluentui/react-next/lib/Toggle';
 import { Stack, IStackStyles } from '@fluentui/react-next/lib/Stack';
 import { memoizeFunction } from '@fluentui/react-next/lib/Utilities';
 import { useBoolean } from '@uifabric/react-hooks';
+import { getEdgeChromiumNoHighContrastAdjustSelector } from '@fluentui/react-next/lib/Styling';
 
 const getStackStyles = memoizeFunction(
   (useTrapZone: boolean): Partial<IStackStyles> => ({
-    root: { border: `2px dashed ${useTrapZone ? '#ababab' : 'transparent'}`, padding: 10 },
+    root: {
+      border: `2px dashed ${useTrapZone ? '#ababab' : 'Window'}`,
```

Comment:
just to point out, this is incorrect as you should not be setting HC styles outside of HC media query.